AWSClientRuntime/Sources/Regions/BundleRegionProvider.swift

```diff
@@ -28,7 +28,7 @@ public struct BundleRegionProvider: RegionProvider {
             return future
         }
 
-        if (region.count > maxSizeRegion) {
+        if region.count > maxSizeRegion {
```

Comment:
I guess we missed this last time, oops -- nonetheless, seems pretty low risk to package this change with this commit.

Reply:
+1